doc/guide/Makefile-guide.am

```diff
@@ -64,31 +62,46 @@ GUIDE_XSLT = \
 	doc/guide/version-greater-or-equal.xsl \
 	$(NULL)
 
-GUIDE_FONTS = $(subst dist/fonts/,doc/guide/html/,$(OPENSANS_FONTS))
+GUIDE_FONTS = $(subst dist/fonts/,dist/guide/html/,$(OPENSANS_FONTS))
 
 EXTRA_DIST += \
 	$(GUIDE_DOCBOOK) \
 	$(GUIDE_INCLUDES) \
 	$(GUIDE_XSLT) \
 	$(GUIDE_FONTS) \
 	$(GUIDE_STATIC) \
-	doc/guide/html \
+	dist/guide/html \
+	dist/guide/links.html \
```

Comment:
Yes, we need to distribute it because it depends on the git state to generate, and we don't want to regenerate it after that point during tarball builds.